detection_rules/devtools.py

```diff
@@ -74,6 +75,86 @@ def build_release(config_file, update_version_lock, release=None, verbose=True):
     return package
 
 
+@dataclasses.dataclass
+class GitChangeEntry:
+    status: str
+    previous_path: Path
```

Comment:
calling this `original_path` might be more accurate

Reply:
thank you, that was bothering me too